README.md

```diff
@@ -73,6 +73,9 @@ Then run the following commands:
 # Create dev database
 poetry run python deploy/devdeploy.py -p local_dev
 
+# Finish local setup 
```

Comment:
```suggestion
# Finish local setup sets internal state to mimic a set-up app (e.g. materializes data, starts tasks)
```

I'm sensitive to this "magical" script which hides that there is additional setup required (e.g. the fact that materialized data isn't actually present when a user installs the app). I'm OK with it to make it easier for developer productivity, but just want to make sure we're clear what the script is doing.